doc/api-specification.yaml

```diff
@@ -112,6 +112,9 @@ paths:
           description: ID of dependency
           required: true
           type: integer
+      operationId: getComments
+      produces:
```

Comment:
This is already defined [here](https://github.com/alpinelab/toolbox/pull/30/files#diff-7af801d6120b8194457e2a609b00234eR21) for the entire specification, you do not need to add it.